draft-bbs-signatures.md

```diff
@@ -680,6 +709,9 @@ octets\_to\_point
 hash\_to\_curve_g1
 : follows the suite defined in (#bls12-381-hash-to-curve-definition-using-shake-256) for the G1 subgroup
 
+hash\_to\_field
```

Comment:
This should have its own `dst` value different but similar to hash_to_curve_g1. I propose `BBS_BLS12381FQ_XOF:SHAKE-256_SSWU_RO`

Reply:
Resolved